packages/donate-button-beta/src/helpers/options-types.ts

```diff
@@ -0,0 +1,54 @@
+import deepMerge, {Options as DeepMergeOptions} from 'deepmerge';
+
+export interface DonateButtonOptions {
+	/**
+	 * Identifier for nonprofit on Every.org; you can get it by going to the
+	 * nonprofit's profile on Every.org and looking at its URL
+	 *
+	 * @example
+	 * If the URL is https://www.every.org/givedirectly, the slug is givedirectly
+	 */
+	readonly nonprofitSlug: string;
+	/**
+	 * Whether or not to render Crypto donation flow instead of the normal
+	 * donation flow
+	 *
+	 * @default false
+	 * For now, only crypto is supported
+	 */
+	readonly crypto: boolean; // For now only this is supported
+}
```

Comment:
This is nowhere near supporting the kinds of stuff we have in `donate-button`, but I wanted to keep it as minimal as possible since that one is huge. We can expand over time.

Also I cut out all the modal stuff, since for now the pressing concern is to get crypto donation embeds working, and this is the quickest way there.